uvicorn/protocols/websockets/wsproto_impl.py

```diff
@@ -65,7 +65,7 @@ def connection_made(self, transport):
 
     def connection_lost(self, exc):
         if exc is not None:
-            self.queue.put_nowait({"type": "websocket.disconnect"})
+            self.queue.put_nowait({"type": "websocket.disconnect", "code": exc.code})
```

Comment:
It's not guaranteed that the exception will have a code. I don't know from where I got that from... It works with Starlette because the exception has a `code` field...
```suggestion
            self.queue.put_nowait({"type": "websocket.disconnect", "code": 1000})
```